src/mixins/blendMixin.ts

```diff
@@ -2,7 +2,7 @@ import { BlendProps } from '../types';
 import { propsAssign } from '../helpers/propsAssign';
 
 export const blendMixin = propsAssign<BlendProps, BlendProps>(
-    ['blendMode', 'effectStyleId', 'effects', 'isMask', 'opacity'],
+    ['blendMode', 'effects', 'isMask', 'opacity', 'effectStyleId'],
```

Comment:
I'm not sure why but the original array (`effectStyleId` before `effects` caused `effectStyleId` to be empty when it was supposed to grab the ID from `props` :( I moved `effectStyleId` to the end of the array and everything started working... 

Just in case, I added a test in [rectangle.test.ts](https://github.com/react-figma/react-figma/pull/500/files#diff-bc23adc59b1a1aac1db5a11f2ba4476257193c9d3afc09594e2552591413c1fd) to make sure `effectStyleId` gets passed in.